src/main/plugins/org.dita.pdf2/xsl/fo/hazard-d.xsl

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+This file is part of the DITA Open Toolkit project. 
+See the accompanying license.txt file for applicable licenses.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:fo="http://www.w3.org/1999/XSL/Format"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:dita-ot="http://dita-ot.sourceforge.net/ns/201007/dita-ot"
+                version="2.0"
+                exclude-result-prefixes="xs dita-ot">
+
+  <xsl:template match="*[contains(@class, ' hazard-d/hazardstatement ')]">
+    <xsl:variable name="number-cells" as="xs:integer" select="2"/>
+    <xsl:apply-templates select="*[contains(@class,' ditaot-d/ditaval-startprop ')]" mode="outofline"/>
+    <fo:table xsl:use-attribute-sets="hazardstatement">
+      <xsl:call-template name="commonattributes"/>
+      <xsl:call-template name="globalAtts"/>
+      <xsl:call-template name="displayAtts">
+        <xsl:with-param name="element" select="."/>
+      </xsl:call-template>
+      <fo:table-column xsl:use-attribute-sets="hazardstatement.image.column"/>
+      <fo:table-column xsl:use-attribute-sets="hazardstatement.content.column"/>
+      <fo:table-body>
+        <fo:table-row keep-with-next="always">
+          <fo:table-cell xsl:use-attribute-sets="hazardstatement.title">
+            <xsl:variable name="atts" as="element()">
+              <xsl:choose>
+                <xsl:when test="@type = 'danger'">
+                  <w xsl:use-attribute-sets="hazardstatement.title.danger"/>
+                </xsl:when>
+                <xsl:when test="@type = 'warning'">
+                  <w xsl:use-attribute-sets="hazardstatement.title.warning"/>
+                </xsl:when>
+                <xsl:when test="@type = 'caution'">
+                  <w xsl:use-attribute-sets="hazardstatement.title.caution"/>
+                </xsl:when>
+                <xsl:otherwise>
+                  <w xsl:use-attribute-sets="hazardstatement.title.notice"/>
+                </xsl:otherwise>
+              </xsl:choose>
+            </xsl:variable>
+            <xsl:sequence select="$atts/@*"/>
+            <fo:block>
+              <xsl:if test="@type = ('danger', 'warning', 'caution')">
```

Comment:
Because it defaults to Caution below - we also check for `empty(@type)`. Current result seems a little odd when not setting `@type` gets me `CAUTION` (no image) but setting `type="caution"` gets me the image + `CAUTION`.